crates/bevy_sprite/src/collide_aabb.rs

```diff
@@ -12,6 +12,8 @@ pub enum Collision {
 
 // TODO: ideally we can remove this once bevy gets a physics system
 /// Axis-aligned bounding box collision with "side" detection
+/// *   a_pos and b_pos are the center positions of the rectangles. (This is different to traditional AABB where the position is the corner)
```

Comment:
```suggestion
/// *   a_pos and b_pos are the center positions of the rectangles, typically obtained by extracting the `translation` field from a `Transform` component
```

Reply:
This adds some more context on how to use this function.

I think the original parenthetical statement is distracting and not terribly helpful once the fields' meaning is documented.